src/chart/cartesianChart.js

```diff
@@ -0,0 +1,211 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.chart.cartesianChart = function(xScale, yScale) {
+
+        xScale = xScale || d3.scale.linear();
+        yScale = yScale || d3.scale.linear();
+
+        var plotAreaMargin = {
+                top: 20,
+                bottom: 30,
+                left: 0,
+                right: 40
+            },
+            yAxisLabel = 'y-axis',
```

Comment:
I can't see anyone leaving these labels as-is, is it worth picking defaults that are more likely to be applicable? E.g. `''`?